src/music/bemidi.cpp

```diff
@@ -34,13 +36,18 @@ void MusicDriver_BeMidi::Stop()
 	midiSynthFile.UnloadFile();
 }
 
-void MusicDriver_BeMidi::PlaySong(const char *filename)
+void MusicDriver_BeMidi::PlaySong(const MusicSongInfo &song)
```

Comment:
BeMidi? How long has this existed? Didn't BeOS support get removed the other week...?

Reply:
Only PSP and WinCE were removed so far.

Reply:
I was going to say drop BeOS, but it's a lot less dead than I assumed.

https://www.haiku-os.org/